src/main/resources/io/jenkins/plugins/designlibrary/Dropdowns/index.jelly

```diff
@@ -24,56 +24,46 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:l="/lib/layout" xmlns:dd="/lib/layout/dropdowns">
-  <s:sample>
-    <p class="jdl-leading-paragraph">${%description.1}</p>
-
-    <p class="jdl-paragraph">${%description.2}</p>
-
-    <div class="jdl-component-group">
-      <div class="jdl-component-sample">
-        <l:overflowButton>
-          <dd:item icon="symbol-star-outline plugin-ionicons-api"
-                   text="${%Action 1}" />
-          <dd:item icon="symbol-star-outline plugin-ionicons-api"
-                   text="${%Action 2}" />
-          <dd:separator />
-          <dd:header text="${%Heading}" />
-          <dd:item icon="symbol-star-outline plugin-ionicons-api"
-                   text="${%Action 3}" />
-          <dd:item icon="symbol-star-outline plugin-ionicons-api"
-                   text="${%Action 4}" />
-          <dd:separator />
-          <dd:submenu icon="symbol-star-outline plugin-ionicons-api" text="${%Submenu}">
+  <s:layout>
```

Comment:
general feedback but pre-existing issue. A dropdown to me is generally what's on the select page, this page is a (sub)type of button

Reply:
That's fair, renamed the page to 'Menu' from 'Dropdowns'.